developer/src/kmc/test/fixtures/basic.txt

```diff
@@ -21,15 +21,15 @@ block(kmxheader)  #  struct COMP_KEYBOARD {
 
   00 10 00 00      #    KMX_DWORD dwFileVersion;  // 0004 Version of the file - Keyman 4.0 is 0x0400
 
-  26 d2 24 03      #    KMX_DWORD dwCheckSum;     // 0008 As stored in keyboard
+  c1 79 c0 dd      #    KMX_DWORD dwCheckSum;     // 0008 As stored in keyboard
```

Comment:
still needed?

Reply:
yes, the kmxplus compiler still generates checksums. We can remove this in another patch.